dashboard/templates/scan/main.html

```diff
@@ -0,0 +1,21 @@
+<!-- Overview for a single scan -->
+{% extends 'base.html' %}
+{% include 'flash.html' %}
+{% block content %}
+
+<div class="container">
+    <div class="row">
+      {% include 'scan/header.html' %}
+    </div>
+    <div class="row">
+      {% if not scan.tag in ['T1','T2'] %}
```

Comment:
Yeah definitely. Moving forward I'd like to add the 'anat', 'dwi', etc. scan types from the config files to the database table and rely on that instead of hard coded tags. I'd have to talk to you guys more about that before doing it though, to make sure there are no annoying complications I'm missing.